nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+reviewed: 2016-03-22
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Throughput on par with MSMQ
```

Comment:
I agree with @johnsimons. Throughput can be similar to MSMQ when SQL Server instance is local to the endpoint and they communicate over shared memory or named pipes transports. When the database is moved to a different node and we start to observe network latency, the performance drops.